cmd/pebble/cmd_validate_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	. "gopkg.in/check.v1"
+
+	pebble "github.com/canonical/pebble/cmd/pebble"
+)
+
+func (s *PebbleSuite) TestValidateCommandYamlOK(c *C) {
+	var err error
+
+	layersDir := filepath.Join(s.pebbleDir, "layers")
+	err = os.Mkdir(layersDir, 0755)
+	c.Assert(err, IsNil)
+
+	fpath := filepath.Join(layersDir, "001-kerncraft.yaml")
```

Comment:
Naming nit: IMO `path` is enough, or `layerPath` (the `f` prefix doesn't really say much). Same in the other test.